jscomp/build_tests/react_ppx/src/alias_default_value_test.res

```diff
@@ -0,0 +1,18 @@
+@@jsxConfig({version: 4})
```

Comment:
Can you add this under `jscomp/test` instead? So it's compiler any time `make lib` is run, and you get normal error messages instead of a failed build test.

It would look like this:
```res
@@bs.config({
  flags: ["-bs-jsx", "4"],
})


module C0 = {
  @react.component
  let make = (~a=2, ~b=a + 2) => {
    React.int(a + b)
  }
}
```

Reply:
To me it gives "The module or file props can't be found.".

Reply:
And same error if I write it in some other project that installs this version of the compiler.

Reply:
Notice `@@bs.config`, as `@@jsxVersion` would not be picked up by a single `bsc` invocation used to run tests.